app/models/job.rb

```diff
@@ -1,4 +1,18 @@
 class Job < ActiveRecord::Base
   validates :title, :level_of_interest, :city, presence: true
   belongs_to :company
+  belongs_to :category
+  has_many :comments, dependent: :destroy
+
+  scope :location, -> (location) {where city: location}
```

Comment:
nice use of scopes 👍 

be sure to have tests though for every method your write in this class